### PR TITLE
Support appearance config

### PR DESCRIPTION
**Pull request by hoyelo**

### CODE REVIEW

File: src/__tests__/__snapshots__/WidgetView.test.tsx.snap [hunk 1]
```diff
@@ -4,6 +4,7 @@ exports[`renders a WidgetView using Snapshots 1`] = `
 <RNWidgetView
   configuration={
     Object {
+      "appearance": 0,
```
Comment (stevenou) [inline src/__tests__/__snapshots__/WidgetView.test.tsx.snap:7]: it's weird to me that the config declaration uses an integer. why can't it use the enum?

Comment (hoyelo) [reply src/__tests__/__snapshots__/WidgetView.test.tsx.snap:7]: @stevenou It's a snapshot generated by Jest. We are indeed using an enum, but Jest will convert it to an integer.

File: android/src/main/java/com/reactnativeshippedsuitesdk/RNWidgetView.kt [hunk 2]
```diff
@@ -68,6 +71,18 @@ class RNWidgetView(context: ThemedReactContext) : FrameLayout(context) {
...
+    if (configuration.hasKey("appearance")) {
+      val appearance = configuration.getInt("appearance")
+      if (appearance != null) {
+        when (appearance) {
+          0 -> config.appearance = ShippedSuiteAppearance.AUTO
+          1 -> config.appearance = ShippedSuiteAppearance.LIGHT
+          2 -> config.appearance = ShippedSuiteAppearance.DARK
+        }
+      }
+    }
```
Comment (stevenou) [inline android/src/main/java/com/reactnativeshippedsuitesdk/RNWidgetView.kt:84]: @hoyelo but why do you need this int -> enum code?

Comment (hoyelo) [reply android/src/main/java/com/reactnativeshippedsuitesdk/RNWidgetView.kt:84]: The declaration of this enum in JavaScript is like this:
```
export enum ShippedSuiteAppearance {
  Auto,
  Light,
  Dark,
}
```
but the enum variable obtained in Android is an integer, so it has to be converted to the enum type defined in Android, which looks like this:
```
enum class ShippedSuiteAppearance(val value: String) {
    LIGHT("light"), DARK("dark"), AUTO("auto");
}
```

Comment (stevenou) [reply android/src/main/java/com/reactnativeshippedsuitesdk/RNWidgetView.kt:84]: hmmm not fully understanding it... but ok

**Review by stevenou — APPROVED** [review]

**Pull request merged**